src/Illuminate/Session/Middleware/StartSession.php

```diff
@@ -44,6 +45,12 @@ public function handle($request, Closure $next)
             return $next($request);
         }
 
+        $sessionCookiePath = config('session.path', '/');
```

Comment:
`$this->manager->getSessionConfig()` is used multiple times inside this middleware to fetch the session config, I think we should be consistent.

Reply:
@X-Coder264 fixed.